### ISSUE TITLE
Code doesn't validate until "Run Code" is pushed several times

### ISSUE THREAD

**Issue by Risle** [FreeCodeCamp/FreeCodeCamp#2949]
Challenge http://www.freecodecamp.com/challenges/waypoint-change-the-css-of-an-element-using-jquery has an issue. Please describe how to reproduce it, and include links to screenshots if possible.

Code fails validation ("Your target1 element should have red text." continues to show a red X even though the condition has been met). Interestingly, if "Run Code" is pushed several (sometimes >5) times, the code does eventually pass even though nothing changed.

Screenshot <em>after</em> pressing "Run Code":
![codecampchallenge](https://cloud.githubusercontent.com/assets/10679949/9600757/ccad476a-5051-11e5-81de-752777838a8b.png)

**Comment by sersy** [comment]
<script>
  $(document).ready(function() {
  
    $("#target1").css("color", "red");
   
  });
</script>

**Comment by bugron** [comment]
@Risle thanks for posting this issue!
I am able to confirm this. I can't pass the test with your code but when I click Run button multiple times, the code passes tests. I tried in FF 38 Incognito mode. But then I tried same code in Chrome 43 Incognito mode and it passed when I first time pushed Run button.
Also [we officially support only Chrome browser](https://github.com/FreeCodeCamp/FreeCodeCamp/issues/2942#issuecomment-136591448).

>***
>Please review the [**Guidelines for Contributing**](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md), thank you!.
>***
